poster/index.html

```diff
@@ -15,64 +15,71 @@
         16. Juli - 19. Juli
     </header>
 
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/3a7tybohjrtaz72/gradient_triangle.png?raw=1" alt="noise-patterns">
-    </figure>
-
-    <section class="primary-details u-text_main">
-        <p> Ausstellung der Studiengänge Kommunikations - design </p>
-        <p> Öffnungszeiten der Ausstellung </p>
-        <ul class="remove-list">
-            <li> 17. Juli 12:00-18:00 </li>
-            <li> 18. Juli 10:00-17:00 </li>
-            <li> 19. Juli 10:00-17:00 </li>
-        </ul>
+    <div class="triangle-pattern"></div>
+
+    <section class="primary-details_flex">
+        <div class="flex-grid u-text_details primary-details">
+            <p> Ausstellung der Studiengänge Kommunikations - design </p>
+            <p> Öffnungszeiten der Ausstellung </p>
+            <ul class="u-remove-list">
+                <li> 17. Juli 12:00-18:00 </li>
+                <li> 18. Juli 10:00-17:00 </li>
+                <li> 19. Juli 10:00-17:00 </li>
+            </ul>
+        </div>
+        <div class="noise_pattern flex-grid">
+        </div>
     </section>
 
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/rfga556mflcpfp5/noise_pattern.png?raw=1" alt="noise-patterns" class="noise-pattern">
-        <!--   <img src= "https://www.dropbox.com/s/xe2evshywwz5dc7/horizontal_stripes.png?raw=1" alt= "noise-patterns" class= "horizontal-pattern" -->
-    </figure>
+    <div class="noise-horizontal_pattern"></div>
 
-    <section class="secondary-details u-text_main">
+    <section class="secondary-details u-text_details">
         <p> Vernissage und Abschlussfeier </p>
         <p> 16. juli </p>
         <p> 18:30 - 21:00 Uhr Eröffnung der Ausstellung </p>
         <p> Ab 22:00 Uhr ist Party. </p>
-        <p> Auftritt der Band >Fiste de pure emociòn< </p> <ul class="remove-list">
+        <p> Auftritt der Band >Fiste de pure emociòn< </p> <ul class="u-remove-list">
                 <li> Dj Biro </li>
                 <li> Dj Wutzekind </li>
                 </ul>
     </section>
 
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/rfga556mflcpfp5/noise_pattern.png?raw=1" alt="noise-patterns">
-    </figure>
-
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/3a7tybohjrtaz72/gradient_triangle.png?raw=1" alt="noise-patterns">
-    </figure>
+    <div class="gradient-triangle_pattern"></div>
 
-    <aside class="initials-texture">
-        <h2>KD</h2>
+    <div class="vertical-dark_noise">
+        <img src="images/vertical_stripes.png" alt="black and white pattern" class="texture vertical-pattern">
```

Comment:
I'm curious about the logic of some of the images (that are not shapes) being inline images vs. background images. Can you explain what led to this?

Reply:
Also! Remember to write your `alt` text more closely like a sentence and end with a period, e.g.:
```suggestion
        <img src="images/vertical_stripes.png" alt="Black and white pattern." class="texture vertical-pattern">
```

Reply:
> I'm curious about the logic of some of the images (that are not shapes) being inline images vs. background images. Can you explain what led to this?

I was originally going to use them all as inline, however, I decided that using them as background caused fewer problems especially when they are paired with other text or other images. It was also easier to switch positions of the images with media queries later on. I only left one image other than the shapes as inline because that one was working the way I wanted to so I didn't see the need to change.